README.md

```diff
@@ -72,7 +72,7 @@ module "vault-ent" {
   # Prefix for resource names
   resource_name_prefix = "dev"
 
-  # SSH public key (for authentication to Vault servers)
+  # SSH public key (needs to be created for authentication to Vault servers)
```

Comment:
I don't love this change, because many users will already have a key to provide. For those that haven't administered linux systems before I do want to improve it to be more clear.

How about leaving the line as is, but adding [this link](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/mac-create-ssh-keys) on the following line so we're directing users to info on how to make the private/public key if they need it?

Reply:
updated!